config.yaml

```
@@ -0,0 +1,58 @@
+    config:
+      base_url: https://api.purpleair.com/v1
+      rate_limit_per_key: 100
+      timeout: 30
+
+  - name: waqi
+    type: waqi
+    enabled: false
+    config:
+      base_url: https://api.waqi.info
+      rate_limit_per_key: 1000
+      timeout: 30
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)